README.md

```diff
@@ -95,6 +95,20 @@ npm test
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Docker
+
+Build docker image
+
+```sh
+docker build -t csaf/validator-service:1.0.0 .
+```
+
+Start container
+
+```sh
+docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service:1.0.0
+```
+
```

Comment:
This is a file to use it in production right? Maybe, we should add that as a subsection under "Getting started" and refer to this in the sentence `To manage the process you can use Docker or an init system of your choice. An example Docker setup is provided below.`

@mfd2007, @Id3aFly : Thoughts?

Reply:
We can 

> > ```shell
> > docker build -t csaf/validator-service:1.0.0 .
> > ```
> 
> Why exactly do we statically reference `1.0.0`?

This was just an example. We can it to `x.y.z` to make it more general. I don't want to use `latest` as tag.

Reply:
> This is a file to use it in production right? Maybe, we should add that as a subsection under "Getting started" and refer to this in the sentence `To manage the process you can use Docker or an init system of your choice. An example Docker setup is provided below.`
> 
> @mfd2007, @Id3aFly : Thoughts?

Would be okay for me.

Reply:
> We can
> 
> > > ```shell
> > > docker build -t csaf/validator-service:1.0.0 .
> > > ```
> > 
> > 
> > Why exactly do we statically reference `1.0.0`?
> 
> This was just an example. We can it to `x.y.z` to make it more general. I don't want to use `latest` as tag.

Is there a special reason why you don't want to use the `latest` tag?

Reply:
`latest` is ambiguous. Let's just skip the tag and write `docker build -t csaf/validator-service .` in the readme.

Reply:
I don't have any preference.